src/index.rs

```diff
@@ -154,6 +159,11 @@ impl IndexWriter {
         let field = get_field(&self.schema, field_name)?;
         let value = extract_value(field_value)?;
         let term = match value {
+            Value::Null => {
+                return Err(exceptions::PyValueError::new_err(format!(
+                    "Field `{field_name}` is null type not deletable."
+                )))
+            },
```

Comment:
Not sure about what to do for deletions - is Err correct here (Null, Array, Object)?

Reply:
Should be fine I think,
The best way of dealing with null types in tantivy-py, when I use it, is to define custom null representations for each dataType, like INT_MAX for integer etc.
Whenever encountering null, I swap out the null value for these custom reprs.
I wonder if we should provide that functionality by default, to bridge the 'null' paradigms in python and rust for tantivy.

Reply:
For now I'm going to keep it as-is, raising an error. We can rethink this in a future PR if necessary.